ep/src/lib.rs

```diff
@@ -1,13 +1,15 @@
-use psm::epid::*;
-use psm::error::*;
-use psm::consts::*;
+extern crate error;
+use error::*;
```

Comment:
Are we using all of the exports from error? I might avoid importing the whole universe unless we're using tons and tons of em.